fdbserver/TLogServer.actor.cpp

```diff
@@ -1506,6 +1513,17 @@ std::deque<std::pair<Version, LengthPrefixedStringRef>>& getVersionMessages(Refe
 	return tagData->versionMessages;
 };
 
+ACTOR Future<Void> waitForMessagesForTag(Reference<LogData> self, TLogPeekRequest const& req) {
```

Comment:
can you add comments on each new function

Reply:
yes, this is on my todo list. I should do a better job writing documentation as I write code, sorry.